recipes/goatools/meta.yaml

```diff
@@ -0,0 +1,26 @@
+about:
+    home: 'https://github.com/tanghaibao/goatools'
+    summary: "Python scripts to find enrichment of GO terms"
+package:
+  name: goatools
+  version: '0.5.9'
+build:
+  number: 0
+  skip: False
+source:
+  fn: goatools-0.5.9.tar.gz
+  md5: 2d5f1f9319dbf59850bf92cfa71e92ee
+  url: https://pypi.python.org/packages/source/g/goatools/goatools-0.5.9.tar.gz
+requirements:
+  build:
+    - python 2.7*
```

Comment:
I guess a better way to do this is to just write `- python` here and above `skip: True # [py3k]`